src/databricks/labs/ucx/assessment/queries/all_tables.sql

```diff
@@ -12,5 +12,7 @@ SELECT `database`,
            ELSE "EXTERNAL"
        END AS storage,
        IF(format = "delta", "Yes", "No") AS is_delta,
-       location
+       location,
+       upgrade_status,
+       upgrade_target
```

Comment:
we cannot have these columns here. this table is overwritten every assessment run and we lose all the state: `df.write.mode("overwrite")`. The only proper way to store it is in the table properties of the source and target table.

Reply:
It doesn't belong to this task/workflow, which is run on a workspace without UC metastore. What are you trying to show?